bridge/elrond/client.go

```diff
@@ -238,7 +238,7 @@ func (c *client) ProposeTransfer(_ context.Context, batch *bridge.Batch) (string
 	hash, err := c.sendTransaction(builder, uint64(proposeTransferCost+len(batch.Transactions)*proposeTransferTxCost))
 
 	if err == nil {
-		c.log.Info("Elrond: Proposed transfer for batch %v with hash %s", batch.Id, hash)
+		c.log.Info(fmt.Sprintf("Elrond: Proposed transfer for batch %v with hash %s", batch.Id, hash))
```

Comment:
please revert. This will display the hash as [123 223 23 23...]

Reply:
Reverted does not look like something we want either.

It will be 
`Elrond: Proposed transfer for batch %v with hash %s" 1 <some hash>`

Will change the text a bit and not use fmt